### ISSUE TITLE
Floating Combat tracker not populated if UI hidden

### ISSUE THREAD

**Issue by AJAnderson** [syl3r86/displaymode#3]
If the combat tracker is popped out via right click, and then UI is hidden by clicking on the anvil, the popped out combat tracker will not be populated when combat starts.

**Comment by AJAnderson** [comment]
Also, it clears itself when the "next combatant" button is hit

**Comment by cytoph** [comment]
Yes, this would be pretty awesome if it gets fixed. Would like to use this for my players to see the current map and combat state on a big TV screen.

A little more detailed description:
It seems that the view is cleared/shown everytime the combat tracker "reloads".
So if the UI is currently visible, the combat has started (therefore is shown in the floating combat tracker) and I click the Foundry icon the content of the floating combat tracker is visible till "next combatant" is clicked.
The other way round, if the floating combat tracker is currently empty due to the UI being hidden and I click the Foundry Icon it keeps being empty - more so even the combat tracker of the side bar is empty. Only if I press "next combatant", both (sidebar and floating combat tracker) than show their content again.
